vkbeautify/vkbeautify-tests.ts

```diff
@@ -0,0 +1,119 @@
+import VKBeautify from 'vkbeautify';
```

Comment:
the only time you need `export = ` is if you are modeling a node module that exports a function. something like yargs for instance.